Endzone.Umbraco.Extensions/PublishedContentExtensions/MultiUrlPicker.cs

```diff
@@ -40,6 +40,41 @@ public static Link GetLink(this IPublishedContent content, string property, bool
         }
 
         /// <summary>
+        /// Gets the links as a collection of IPublishedContent. Check for null.
```

Comment:
Sorry this slipped my attention earlier: The comment says "Check for null" but it's safe here - you always return a list.